lib/Agrammon/Documentation.pm6

```diff
@@ -0,0 +1,111 @@
+use Agrammon::Model;
+
+#| Create LaTeX document of Model
+sub prepare-model( Agrammon::Model $model, :%technical! ) is export {
```

Comment:
If `create-latex` did not produce output you could call it in the test, and then probably not export this implementation detail.

Reply:
... making this trivial. Done.